pyproject.toml

```diff
@@ -35,17 +35,22 @@ dev = [
 ]
 
 [build-system]
-requires = ["setuptools"]
-build-backend = "setuptools.build_meta"
+requires = ["meson @ git+https://github.com/moi15moi/meson.git@fix", "meson-python", "pybind11==2.13.6"]
```

Comment:
I'm waiting for meson to fix this problem: https://github.com/mesonbuild/meson/pull/13916
Currently, I set a branch that has the fix.

I'm also waiting pybind11 to fix this issue with the version 3.0.0: https://github.com/pybind/pybind11/discussions/5750



 and pybind11